mollie_subscription_ept/controllers/main.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+import logging
+import werkzeug
+from odoo import fields, http
+from odoo.http import request, Response
+from odoo.tools.json import scriptsafe as json_scriptsafe
+
+_logger = logging.getLogger(__name__)
+
+
+class WebsiteSale(http.Controller):
+
+    @http.route(['/clear_warn'], type='json', auth="public", website=True)
+    def clear_warning(self, **post):
+        hide_warn = post.get('hide_warn')
+        if hide_warn:
+            request.session['show_wishlist_warn'] = request.session['show_in_cart_warn'] = False
+        return True
+
+    def _set_session(self, so_obj):
+        request.session['show_wishlist_warn'] = True if so_obj.get('show_wishlist_warning') else False
+        request.session['show_in_cart_warn'] = True if so_obj.get('show_in_cart_warning') else False
+
+    @http.route(['/shop/cart/update'], type='http', auth="public", methods=['POST'], website=True)
+    def cart_update(self, product_id, add_qty=1, set_qty=0, **kw):
```

Comment:
The function should not be overriden, make use of super call, else it may affect the working of other modules